packages/sdk/client/src/tests/indexing.test.ts

```diff
@@ -236,8 +260,9 @@ describe('Index queries', () => {
     const client = await initClient(builder.createLocalClientServices());
     await client.halo.createIdentity();
     const space = await client.spaces.create();
-    const contact = await addContact(space, john);
-    const document = await addDocument(space, 'important document');
+
+    const [contact] = await addObjects(space, getObjects().contacts);
```

Comment:
this generates all of the data every time.
better to generate the data once -- and generate from static arrays of RAW object.
but at least
```
const { contacts, documents } = createObjects();
```